lib/service/storage/elasticsearch.js

```diff
@@ -391,38 +406,47 @@ class ElasticSearch extends Service {
         body.remaining = body.hits.total.value - body.hits.hits.length;
       }
 
-      return this._formatSearchResult(body);
+      return await this._formatSearchResult(body);
     }
     catch (error) {
       throw this._esWrapper.formatESError(error);
     }
   }
 
-  _formatSearchResult (body) {
-    function formatHit (hit) {
+  async _formatSearchResult (body) {
+    
+    const formatHit = async (hit) => {
```

Comment:
I agree but I'm not sure how to refactor that since `_getAliasFromIndice` is making a request to ES and I cannot cache them since Aliases can be updated to point to another index